subprojects/core/src/main/java/org/gradle/testfixtures/internal/ProjectBuilderImpl.java

```diff
@@ -84,8 +84,8 @@ public Project createProject(String name, File inputProjectDir, File gradleUserH
         startParameter.setGradleUserHomeDir(userHomeDir);
         NativeServices.initialize(userHomeDir);
 
-        BuildExecutionTimer buildExecutionTimer = BuildExecutionTimer.startingNow();
-        BuildSessionScopeServices buildSessionScopeServices = new BuildSessionScopeServices(getUserHomeServices(userHomeDir), startParameter, buildExecutionTimer, ClassPath.EMPTY);
+        BuildStartedTime buildStartedTime = BuildStartedTime.startingAt(System.currentTimeMillis());
```

Comment:
Because this is _kind of_ a clock sync point, or the closest we have to it in this circumstance.

This class is used to build `Project` objects used for unit testing. You can actually use it to run builds. Moreover, there's no meaningful prior sync point, or subsequent event that really needs guaranteed montonicity.